app/cafe.py

```diff
@@ -0,0 +1,20 @@
+import datetime
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("Visitor should be vaccinated")
+        if datetime.date.today() > visitor["vaccine"]["expiration_date"]:
+            raise OutdatedVaccineError("Vaccine is outdated!")
+        if not visitor["wearing_a_mask"]:
```

Comment:
I'd write `elif`, but hope that won't be a big problem)

Reply:
It won't be problem here, since we will rise an Error, but in future it could be some blocks which we are going to change, so better to adhere overall concese logic.